### PR TITLE
Change wording on EXTENDED_VALIDATION env variable

### PR DESCRIPTION
**Pull request by pecameron**
Openshift 3.6

The new wording better describes what is being done.

### CODE REVIEW

**Comment by pecameron** [comment]
@knobunc @bfallonf PTAL

File: architecture/core_concepts/routes.adoc [hunk 1]
```diff
@@ -253,7 +253,7 @@ $ oc set env dc/router ROUTER_SYSLOG_ADDRESS=127.0.0.1 ROUTER_LOG_LEVEL=debug
 |`*DEFAULT_CERTIFICATE*` |  | The contents of a default certificate to use for routes that don't expose a TLS server cert; in PEM format.
 |`*DEFAULT_CERTIFICATE_DIR*` |  | A path to a directory that contains a file named *_tls.crt_*. If *_tls.crt_* is not a PEM file which also contains a private key, it is first combined with a file named tls.key in the same directory. The PEM-format contents are then used as the default certificate. Only used if `DEFAULT_CERTIFICATE` or `DEFAULT_CERTIFICATE_PATH` are not specified.
 |`*DEFAULT_CERTIFICATE_PATH*` |  | A path to default certificate to use for routes that don't expose a TLS server cert; in PEM format. Only used if `DEFAULT_CERTIFICATE` is not specified.
-|`*EXTENDED_VALIDATION*` | `true` | If `true`, perform an additional extended validation step on all routes admitted by this router.
+|`*EXTENDED_VALIDATION*` | `true` | If `true`, the router confirms the certificates can be verified. Set false to turn off the tests.
```
Comment (bfallonf) [inline architecture/core_concepts/routes.adoc]: s/false/`false`

(backticks)

**Comment by bfallonf** [comment]
Thanks @pecameron . Just one nit, but otherwise all good. Let me know when you've taken a look and I can merge.

File: architecture/core_concepts/routes.adoc (hunk 1 above)
Comment (knobunc) [inline architecture/core_concepts/routes.adoc]: the router confirms that the certificate is structurally correct.  It does not verify the certificate against any CA.

**Comment by pecameron** [comment]
@bfallonf @knobunc PTAL

**Comment by bfallonf** [comment]
Thanks @pecameron . I'm happy with this if everyone else is. Is this for a BZ or trello card?

**Comment by pecameron** [comment]
@bfallonf No trello card of BZ, came out of discussion on sme list (we were vague about what was happening)

File: architecture/core_concepts/routes.adoc [hunk 2]
```diff
@@ -253,7 +253,7 @@ $ oc set env dc/router ROUTER_SYSLOG_ADDRESS=127.0.0.1 ROUTER_LOG_LEVEL=debug
 |`*DEFAULT_CERTIFICATE*` |  | The contents of a default certificate to use for routes that don't expose a TLS server cert; in PEM format.
 |`*DEFAULT_CERTIFICATE_DIR*` |  | A path to a directory that contains a file named *_tls.crt_*. If *_tls.crt_* is not a PEM file which also contains a private key, it is first combined with a file named tls.key in the same directory. The PEM-format contents are then used as the default certificate. Only used if `DEFAULT_CERTIFICATE` or `DEFAULT_CERTIFICATE_PATH` are not specified.
 |`*DEFAULT_CERTIFICATE_PATH*` |  | A path to default certificate to use for routes that don't expose a TLS server cert; in PEM format. Only used if `DEFAULT_CERTIFICATE` is not specified.
-|`*EXTENDED_VALIDATION*` | `true` | If `true`, perform an additional extended validation step on all routes admitted by this router.
+|`*EXTENDED_VALIDATION*` | `true` | If `true`, the router confirms that the certificate is structurally correct. It does not verify the certificate agains any CA. Set `false` to turn off the tests.
```
Comment (bfallonf) [inline architecture/core_concepts/routes.adoc]: s/agains/against

**Comment by bfallonf** [comment]
Thanks @pecameron . There's a small typo, but once that's fixed, I'll merge.

File: architecture/core_concepts/routes.adoc (hunk 2 above)
Comment (pecameron) [inline architecture/core_concepts/routes.adoc]: fixed

**Comment by pecameron** [comment]
@bfallonf PTAL

**Comment by bfallonf** [comment]
Thanks for that @pecameron . I'll merge away.

I don't think a review or rev history is needed.

**Pull request merged**